content/applications/finance/payment_providers/xendit.rst

```diff
@@ -0,0 +1,45 @@
+======
+Xendit
+======
+
+`Xendit <https://www.xendit.co>`_ is an Indonesian-based payment solution provider that covers
+several Southeast Asian countries. It allows businesses to accept credit cards as well as several local
+payment methods.
+
+.. _payment_providers/xendit/configure_dashboard:
+
+Configuration on the Xendit Dashboard
+=====================================
+
+#. Create a Xendit account if necessary and log in to the `Xendit Dashboard
+   <https://dashboard.xendit.co>`_.
+#. Check your account mode in the top left corner of the page. Use the :guilabel:`Test Mode` to try the integration
+   without charging your customers. Switch to :guilabel:`Live Mode` once you are ready to accept
+   payments.
+#. Navigate to :menuselection:`Configuration: Settings` in the left part of the application page. In the :guilabel:`Developers` section,
+   click `API Keys <https://dashboard.xendit.co/settings/developers#api-keys>`_. In tis page, you will setup for both :guilabel:`API key`
+   and :guilabel:`webhook token`.
```

Comment:
I don't think the second part of the sentence is necessary. I think it's fine if users follow the rest of the procedure and find out what they need to when they get to it?
If you really want to keep it, then I would change it to:

This page allows you to set up both the API key and webhook token. 
(no guilabel, but you could use bold if you want to)